core/src/main/java/hudson/model/Queue.java

```diff
@@ -1928,11 +1929,6 @@ default CauseOfBlockage getCauseOfBlockage() {
          */
         String getName();
 
-        /**
-         * @see hudson.model.Item#getFullDisplayName()
-         */
-        String getFullDisplayName();
-
         /**
          * Returns task-specific key which is used by the {@link LoadBalancer} to choose one particular executor
          * amongst all the free executors on all possibly suitable nodes.
```

Comment:
We might want to switch this to `getFullName` if `this instanceof FullyNamed`.

https://github.com/jenkinsci/workflow-durable-task-step-plugin/blob/1b595c29ddd712c7f1ea2b30c4356d979e4a67bc/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java#L838-L845 is the other implementation.

Reply:
Looks reasonable.